src/MartinGeorgiev/Doctrine/DBAL/Types/BooleanArray.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+/**
+ * Implementation of PostgreSql BOOL[] data type.
+ *
+ * @see https://www.postgresql.org/docs/9.4/static/arrays.html
+ * @since 1.5.3
+ *
+ * @author Martin Georgiev <martin.georgiev@gmail.com>
```

Comment:
You can include your authorship details here.

Reply:
Done, thanks!

Reply:
```suggestion
```